tests/Authorization.AspNetCore.Tests/ValidationTestBase.cs

```diff
@@ -92,7 +93,8 @@ private IValidationResult Validate(ValidationTestConfig config)
             Document = document,
             Operation = document.Definitions.OfType<GraphQLOperationDefinition>().First(),
             Rules = config.Rules,
-            Variables = config.Variables
+            Variables = config.Variables,
+            RequestServices = ServiceProvider,
```

Comment:
```suggestion
            RequestServices = ServiceProvider, // root provider here instead of scoped one, but since nothing registered is scoped, it makes a little difference so we just use rooot provider directly
```